internal/transform/transform.go

```diff
@@ -101,6 +102,14 @@ func AddComponentProps(doc *tycho.Node, n *tycho.Node) {
 	}
 }
 
+func CollectExternalResources(doc *tycho.Node, n *tycho.Node) {
+	if n.Type == tycho.ElementNode && n.DataAtom == a.Link {
+		if len(n.Attr) > 0 {
+			doc.ExternalResources = append(doc.ExternalResources, n.Attr)
```

Comment:
Oh good idea. I‘ll see how big of a change that is

Reply:
Updated. I left `Styles` and `Scripts` at the top-level doc, but moved everything else inside Metadata. Very quick change to make. (note: even though _some_ scripts go inside metadata, I left it out because it’s half-in, half-out)